web/api/Makefile

```diff
@@ -1,5 +1,13 @@
 THRIFT_OPTS = -r -o /data
-THRIFT_VERSION = "0.12.0"
+
+# The default for THRIFT_DOCKER_IMAGE is the official thrift image
+THRIFT_DOCKER_IMAGE ?= thrift
+# The current default for THRIFT_VERSION is latest version of the currently
+# unmaintained official thrift image on dockerhub
+# This is the version of the Docker image, and currently we are operating on
```

Comment:
This statement about version matching is just too speculative and refer to a bigger context, so I remove them.